client/network/src/service.rs

```diff
@@ -61,6 +61,7 @@ use sp_runtime::{
 	ConsensusEngineId,
 };
 use sp_utils::mpsc::{tracing_unbounded, TracingUnboundedReceiver, TracingUnboundedSender};
+use sp_core::offchain::OffchainStorage;
```

Comment:
I'd strongly prefer is the network didn't depend on some offchain storage types. Let's please not spaghetti-fy the code even more.